scenario_configs/librispeech_asr_snr_targeted.json

```diff
@@ -0,0 +1,81 @@
+{
+    "_description": "Baseline DeepSpeech ASR on LibriSpeech, contributed by MITRE Corporation",
+    "adhoc": {
+        "skip_adversarial": false
+    },
+    "attack": {
+        "knowledge": "white",
+        "kwargs": {
+            "batch_size": 1,
+            "eps": 10,
+            "eps_step": 0.5,
+            "max_iter": 10,
+            "norm": "snr",
+            "num_random_init": 0,
+            "targeted": true
+        },
+        "module": "armory.art_experimental.attacks.snr_pgd",
+        "name": "SNR_PGD_Numpy",
+        "targeted": true,
+        "targeted_labels": {
+            "kwargs": {
+                "import_from": "armory.attacks.librispeech_target_labels",
+                "transcripts": "matched_length"
+            },
+            "module": "armory.utils.labels",
+            "name": "MatchedTranscriptLengthTargeter"
+        },
+        "use_label": false
+    },
+    "dataset": {
+        "batch_size": 1,
+        "eval_split": "test_clean",
+        "framework": "numpy",
+        "module": "armory.data.datasets",
+        "name": "librispeech",
+        "train_split": "train_clean100"
+    },
+    "defense": null,
+    "metric": {
+        "means": false,
+        "perturbation": "linf",
+        "record_metric_per_sample": true,
+        "task": [
+            "word_error_rate"
```

Comment:
We didn't have a config that includes targeted WER with the full set of the targeted labels.
Since I was putting these labels into a file with the ground truth and entailment, I wanted to add a config to show the new usage.